ee/server/scim/users.go

```diff
@@ -260,6 +265,8 @@ func createUserResource(user *fleet.ScimUser) scim.Resource {
 }
 
 // GetAll
+// Pagination is 1-indexed.
```

Comment:
Is a requirement from the SCIM RFC? If so, would you please add a quick note/link for future readers.

Based on the test usage, I'm confused whether the index refers to the index of the list element or the index of the page. At first I thought that it was meant to be the index of the page (assuming it was like pagination for other Fleet endpoints), but seeing more example, it looks like it is the actually index of the element.